wordpress-mobile/AztecEditor-Android
# Improve nested lists behaviour

### Fix
The nested lists were not previously working properly. In this PR I'm fixing the behaviour. 
- the clicked nested list is now correctly highlighted in the toolbar
- you can change type of the selected nested list (without changing the parent list type)
- you can change type only of the parent list type, not of the child
- you can change type of both if you select both


### Test
1. Call `toolbar.setToolbarItems(ToolbarItems.BasicLayout(ToolbarAction.LIST, ToolbarAction.INDENT, ToolbarAction.OUTDENT))` in the `MainActivity` to enable indents and list operations
2. Run the app
3. Add a list
4. Try toggling its type
5. Add nested lists by `indent` 
6. Try toggling type of the nested list
7. Try toggling type of the parent list
8. Click around and notice the toolbar `list` icon is highlighted according to the selected list

### Review
@AmandaRiu 

Make sure strings will be translated:


https://user-images.githubusercontent.com/1079756/163550607-316bfc3b-5595-4f39-bc3d-3a04b19f38b4.mp4
